README.md

```diff
@@ -58,6 +58,9 @@ import { init, parse } from 'es-module-lexer/dist/es-module-lexer.js';
   // Returns "asdf"
   source.substring(imports[0].s, imports[0].e);
 
+  // Returns "import { a } from 'asdf';"
+  source.substring(imports[0].ss, imports[0].se);
```

Comment:
can't comment outside the diff, but below here I saw that you're using `d`  for what is now statement_start. This works because we were already storing this value as the flag value for whether this import is dynamic or not. 

```
// Returns "import /*comment!*/ ("
  source.substring(imports[1].d, imports[1].s);	  source.substring(imports[1].d, imports[1].s);
```

Now that we have a true statement start/end value for standard imports, should we convert `dynamic` to `true` (or some other const/enum value) and then include statement start/end for dynamic imports as well?

Reply:
That could well make sense... although worth noting this is a breaking change. Could be better to just double up for now then make the break later on.

Reply:
works for me, I'll leave that potential cleanup for a later PR.